.github/workflows/ci-workflow.yml

```diff
@@ -218,6 +218,36 @@ jobs:
       with:
         name: premake-openbsd-${{ matrix.platform }}
         path: bin/${{ matrix.config }}/
+  netbsd:
```

Comment:
Possibly
```
defaults:
  run:
    shell: netbsd {0}
    working-directory: $GITHUB_WORKSPACE
```
[setting-default-shell-and-working-directory](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/setting-a-default-shell-and-working-directory#setting-default-shell-and-working-directory)

Reply:
I'll investigate this, but I think it's best to leave this PR as-is. It's currently consistent with FreeBSD and OpenBSD jobs, and it would be out of scope of this PR to modify them to use these defaults.

Reply:
I've investigated, only the `shell` option worked out. The VM actions look like they don't support the `working-directory` option with the shell they setup. I've got a few more PRs to do before I'll submit the PR to move to defaulting the shell for these CI jobs.